src/pages/JobLogs.tsx

```diff
@@ -1,18 +1,49 @@
+import { useQuery } from "@apollo/client";
 import styled from "@emotion/styled";
 import Button from "@leafygreen-ui/button";
-import { Body, H2, Link } from "@leafygreen-ui/typography";
+import { Body, H2, H3, Link } from "@leafygreen-ui/typography";
 import { useParams } from "react-router-dom";
-import { PageWrapper } from "components/styles";
+import { PageWrapper, StyledRouterLink } from "components/styles";
+import { TablePlaceholder } from "components/Table/TablePlaceholder";
+import { getParsleyBuildLogURL } from "constants/externalResources";
+import { getTaskRoute } from "constants/routes";
 import { size } from "constants/tokens";
+import { useToastContext } from "context/toast";
+import {
+  LogkeeperBuildMetadataQuery,
+  LogkeeperBuildMetadataQueryVariables,
+} from "gql/generated/types";
+import { GET_LOGKEEPER_BUILD_METADATA } from "gql/queries";
 import { usePageTitle } from "hooks";
 import { environmentalVariables } from "utils";
+import { JobLogsTable } from "./jobLogs/JobLogsTable";
 
 const { isProduction } = environmentalVariables;
 
 export const JobLogs = () => {
   const { buildId } = useParams<{ buildId: string }>();
+  const dispatchToast = useToastContext();
+
   usePageTitle(`Job Logs - ${buildId}`);
 
+  const { data } = useQuery<
+    LogkeeperBuildMetadataQuery,
+    LogkeeperBuildMetadataQueryVariables
+  >(GET_LOGKEEPER_BUILD_METADATA, {
+    variables: { buildId },
+    onError: (err) => {
+      dispatchToast.error(
+        `There was an error retrieving logs for this build: ${err.message}`
+      );
+    },
+  });
+
+  if (!data) return null;
```

Comment:
Should we show a loading state?

Reply:
I was imagining using a skeleton and the 404 graphic to represent loading and not found like we do for the task and version pages.